templates/taxi/driver_list.html

```diff
@@ -1,6 +1,10 @@
 {% extends "base.html" %}
-
+{% load crispy_forms_filters %}
 {% block content %}
+  <form action="" method="get" class="form-inline">
+  {{ search_form|crispy }}
+  <input class="btn btn-secondary" type="submit" value="🔎">
+  </form>
```

Comment:
Move to a separate template and include where needed

Reply:
Done